packages/ses/README.md

```diff
@@ -1,7 +1,12 @@
-# Secure ECMAScript (SES)
+# SES
 
-Secure ECMAScript (SES) is an execution environment that provides fine-grained
-sandboxing with Compartments.
+SES is *hardened JavaScript*. SES stands for *fearless cooperation*.
+This package is a SES [shim](https://en.wikipedia.org/wiki/Shim_(computing))
+for JavaScript features [proposed](https://github.com/tc39/proposal-ses) to
+ECMA TC-39.
+Hardened JavaScript is highly compatible with ordinary JavaScript, so a
+surprising quantity of existing JavaScript libraries can run on hardened
```

Comment:
Why is it surprising to this audience? Where did they get other expectations about SES?